docs/spark/how-to-guides/ubuntu-instructions.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Build a .NET for Apache Spark application on Ubuntu
+description: Learn how to build your .NET for Apache Spark application on Ubuntu
+ms.date: 01/02/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+
+# Learn how to build your .NET for Apache Spark application on Ubuntu
+
+## Prerequisites
+
+If you already have all the prerequisites, skip to the [build](#build) steps below.
+
+  1. Download and install **[.NET Core 2.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/2.1)** or the **[.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1)** - installing the SDK adds the `dotnet` toolchain to your path.
```

Comment:
Are only .NET Core 2.1 and 3.1 supported (or recommended) for this? If so, this sounds great. If other versions of .NET Core are also compatible, might be worth noting which versions are supported/recommended.

Reply:
Looking ahead, it looks like the version of this line in the Windows how-to guide lists the specifically-supported versions. Would we like to list the same here?

Reply:
Makes sense, will make the change.